packages/charts-components/templates/angular/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "charts-components-angular-template",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "yarn copy && ng serve",
+    "build": "yarn copy && ng build",
+    "watch": "yarn copy && ng build --watch --configuration development",
+    "test": "ng test",
+    "copy": "copy-folder ./node_modules/@arcgis/charts-components-angular ./charts-components-angular"
```

Comment:
Ditto, if this can be done via CDN.

Reply:
We had some issues testing locally specifically with the Angular template. Tried multiple approaches, loading from node_module, CDN. Nothing worked but the local copy-folder way. 

I will continue looking into this, and will update if we can use CDN or some other way.